app.py

```diff
@@ -63,21 +63,25 @@ def run_app(self):
             message += "in and out frame data for this Shot."
             QtGui.QMessageBox.information(None, "No data in Shotgun!", message)
             return
-            
+
         # now update the frame range.
         # because the frame range is often set in multiple places (e.g render range,
         # current range, anim range etc), we go ahead an update every time, even if
         # the values in Shotgun are the same as the values reported via get_current_frame_range()
-        self.set_frame_range(self.engine.name, new_in, new_out)
-        
-        message =  "Your scene has been updated with the \n"
-        message += "latest frame ranges from shotgun.\n\n"
-        message += "Previous start frame: %s\n" % current_in
-        message += "New start frame: %s\n\n" % new_in
-        message += "Previous end frame: %s\n" % current_out
-        message += "New end frame: %s\n\n" % new_out
-        
-        QtGui.QMessageBox.information(None, "Frame range updated!", message)
+        updated = self.set_frame_range(self.engine.name, new_in, new_out)
```

Comment:
I think rather than get a return for success/failure here, it would be good to just catch and handle an exception if one is raised -- as @manneohrstrom suggested, we can make it the expectation that the hook will succeed or raise an exception if it does not.  The contents of that exception could be used in the warning messagebox below and we could also log the traceback to help with debugging.

Reply:
Cool done!